statute.md

```diff
@@ -10,101 +10,126 @@ title: Berlin TWC Statute
 1.1
 
 This document defines:
-* Decision making process of Berlin Tech Workers Coalition (Berlin TWC from now on)
-* How to become a member, and lose membership
-* Working Teams: scope, creation process, disband process
-* Credentials management and sharing
+* The decision-making process of Berlin Tech Workers Coalition (Berlin TWC)
+* How to receive and lose membership
+* Working teams: scope, creation process, disband process
+* Credentials: management and sharing
 
 1.2
 
-The Statute exists to foster participation in Berlin TWC by creating a transparent and democratic decisional and organizational framework. 
+The statute exists to foster participation in Berlin TWC by creating a transparent and democratic decision making framework.
  
 1.3
-Berlin TWC has as its primary goal the empowerment of Tech Workers in their industry and in the society at large, in solidarity with other workers. Power here is understood as the capability of collectives to shape their own companies, governments. This goal will be pursued through workers’ organization, workers’ education and the progressive liberation of technological production.
+The primary goal of Berlin TWC is the empowerment of Tech Workers in their industry and in the society at large, in solidarity with other workers. Power here is understood as the capability of collectives to shape their own companies, governments. This goal will be pursued through workers’ organisation, workers’ education and the progressive liberation of technological production.
 
 
-The application and modification of this document must be driven by the principles of fairness, horizontality, inclusivity organizational scalability and safety of the members. 
+The application and modification of this document must be driven by the principles of fairness, horizontality, inclusivity, organisational growth and safety of the participants. 
 
 
+1.3.1
+
+The members of Berlin TWC should be considered equal among each other and equal to every other worker. A member is expected to show solidarity to other workers through the pursuit of the organisation's goals. Each member contributes at the best of their intellectual, physical and economic capabilities.  
```

Comment:
Suggestion: Replace `Each member contributes at the best of their intellectual, physical and economic capabilities.  ` with `Every member is expected to contribute to best of their knowledge and strength to the best of their ability`